pkg/go/graph/graph_test.go

```diff
@@ -0,0 +1,87 @@
+package graph
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/require"
+
+	language "github.com/openfga/language/pkg/go/transformer"
+)
+
+func TestReverseGraph(t *testing.T) {
+	t.Parallel()
+	testCases := map[string]struct {
+		model          string
+		expectedOutput string // can visualize in https://dreampuf.github.io/GraphvizOnline
+	}{
+		`direct_assignment`: {
+			model: `
+				model
+					schema 1.1
+				type user
+				type company
+					relations
+						define member: [user]
+						define owner: [user]
+						define approved_member: member or owner
+				type group
+					relations
+						define approved_member: [user]
+				type license
+					relations
+						define active_member: approved_member from owner
+						define owner: [company, group]`,
+			expectedOutput: `digraph {
```

Comment:
Does it make sense to have a compare of with and without Reverse - we don't have the dot diagram in the unit test for the above model without reverse

Reply:
Maybe we can also have `authorization-model.reverse.dot` for tests, same as we have 
https://github.com/openfga/language/blob/main/tests/data/transformer/02-type-single-relation/authorization-model.dot ?

Reply:
I thought about that, but it's soooo much work and i'm lazy 😆  would you be OK if in the existing tests i assert that `dot(reverse(reverse(graph)) == dot` ?

Reply:
Updated @rhamzeh @adriantam